### PR TITLE
docs(aws): slugbuild on fargate guide

### PR DESCRIPTION
**Pull request by robinmonjo**
Advanced guide for running slugbuilds on Fargate when using EKS. Happy to take feedback 😊

### CODE REVIEW

File: src/managing-workflow/aws-eks-running-builds-on-fargate.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+3. Apply the limit range to the `deis-builder` namespace using the following command:
+
+```shell
+kubectl apply -f limit-range.yaml
+```
+
+This will set the default capacity for all containers in the `deis-builder` namespace, including the `slugbuild` pods.
+
+
+## Cleanup operator
```
Comment (Cryptophobia) [inline src/managing-workflow/aws-eks-running-builds-on-fargate.md:209]: @robinmonjo , thank you for the guide and contribution .

Looks like there is a duplicate section about the cleanup-operator here? Seems like lines 209-226 are equivalent to the lines after those?
https://github.com/teamhephy/workflow/pull/151/files#diff-636490027cfc34ec7049a941bd713f01a0edbbe9923f97bbe56795af05aa3160R209-R226

Comment (robinmonjo) [reply src/managing-workflow/aws-eks-running-builds-on-fargate.md]: Yes sorry, I did not remove the initial draft version of this part. Good now 😊

**Comment by Cryptophobia** [comment]
lgtm!

**Review by Cryptophobia — APPROVED** [review]
lgtm!

**Pull request merged**